pkg/generate/app/cmd/newapp.go

```diff
@@ -559,9 +550,37 @@ func (c *AppConfig) RunQuery() (*QueryResult, error) {
 	}, nil
 }
 
+// envInCmdLine checks if the variable exists in commandline.
+func envInCmdLine(envVarName string, envVarNameList []string) bool {
```

Comment:
call this "envInList" (this function doesn't care if the value came from the command line)